static/css/components/search-result-item.less

```diff
@@ -56,7 +55,11 @@
   span.resultType {
     font-size: .6875em;
   }
+  .publishedYear{
+    display:block;
+  }
   .preview-covers {
+    display:block;
```

Comment:
please keep whitespace consistent with the rest of the file for readability:
space after `:`